corehq/apps/userreports/templates/userreports/report_error.html

```diff
@@ -3,14 +3,14 @@
 {% load i18n %}
 {% load compress %}
 
-{% block title %}{{ report.title|default:"Project Reports"|safe }}{% endblock %}
+{% block title %}{{ report.title|default:"Project Reports" }}{% endblock %}
 
 {% block page_title %}{% endblock %}
 
 {% block page_content %}
   {% block report_alerts %}
     <div id="report-error" class="alert alert-danger">
-      <p>{{ error_message|safe }}</p>
+      <p>{{ error_message }}</p>
```

Comment:
How do you know there aren't error messages that contain HTML? Anecdotally, I thought we sometimes included a link to report an issue in reporting error messages.

Reply:
This appears to be a pretty narrow template. `report_error.html` is only referenced in `ConfigureReport.dispatch()` and `ConfigurableReportView.get()`. In both circumstances, the error messages are contained directly in the code, and none of them contain HTML. See: [ConfigureReport](https://github.com/dimagi/commcare-hq/blob/56fc7fc6767bf93d95b0e969ec36ac2038e8267a/corehq/apps/userreports/views.py#L525-L531) and [ConfigurableReportView](https://github.com/dimagi/commcare-hq/blob/56fc7fc6767bf93d95b0e969ec36ac2038e8267a/corehq/apps/userreports/reports/view.py#L300-L315). Theoretically, `main_context` could override `error_message` afterwards, but I see no instances of it doing so.

Reply:
cool, thanks for the due diligence